packages/flutter/lib/src/rendering/sliver_grid.dart

```diff
@@ -314,7 +314,7 @@ abstract class SliverGridDelegate {
 /// {@tool dartpad}
 /// Here is an example using the [childAspectRatio] property. On a device with a
 /// screen width of 800.0, it creates a GridView with each tile with a width of
-/// 200.0 and a height of 100.0.
+/// 200.0 and a height of 400.0.
```

Comment:
Good catch, but I think 100 was the intended result and that we should fix the example (by setting `childAspectRatio` to 2 instead of 0.5). Using 2 also makes the example more understandable (easier to understand that the height is equals to width / childAspectRatio).

Reply:
Ok sure, I changed it in [docs: Use 2 for childAspectRatio](https://github.com/flutter/flutter/pull/153500/commits/59e27ab37e62ab66e5b4518ee65177f08cb58a56)